docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 14
+title: ADR Template
+---
+# ADR 013: Slashing on the provider for consumer equivocation
+
+## Changelog
+* 1st Sept. 2023: Initial draft
+
+## Status
+Proposed
+
+## Context
+This ADR presents some approaches on how to slash on the provider chain validators that performed equivocations on consumer chains.
+Currently, the provider chain can [receive and verify evidence of equivocation](https://github.com/cosmos/interchain-security/pull/1232), but it cannot slash the misbehaving validator.
+
+In the remainder of this section, we explain how slashing is performed on a single chain and show why slashing on the provider for equivocation on the consumer is challenging.
+
+Note that future versions of the Cosmos SDK, CometBFT, and ibc-go could modify the way we slash, etc. Therefore, a future reader of this ADR, should note that when we refer to Cosmos SDK, CometBFT, and ibc-go we specifically refer to their [v0.47](https://docs.cosmos.network/v0.47/intro/overview),  [v0.37](https://docs.cometbft.com/v0.37/) and [v7.3.0](https://github.com/cosmos/ibc-go/blob/v7.3.0) versions respectively.
+
+### Single-chain slashing
+Slashing is implemented across the [slashing](https://docs.cosmos.network/v0.47/modules/slashing)
+and [staking](https://docs.cosmos.network/v0.47/modules/staking) modules. 
+The slashing module's keeper calls the staking module's `Slash()` method, passing among others, the `infractionHeight` (i.e., the height when the equivocation occurred), the validator's `power` at the infraction height, and the `slashFactor` (currently set to `5%` in case of equivocation on the Cosmos Hub).
+
+#### Slashing undelegations and redelegations
+To slash undelegations, `Slash` goes through all undelegations and checks whether they started before or after the infraction occurred. If an undelegation started before the `infractionHeight`, then it is **not** slashed, otherwise it is slashed by `slashFactor`.
+
+The slashing of redelegations happens in a similar way, meaning that `Slash` goes through all redelegations and checks whether the redelegations started before or after the `infractionHeight`.
+
+#### Slashing delegations
+Besides undelegations and redelegations, the validator's delegations need to also be slashed.
+This is performed by deducting the appropriate amount of tokens from the validator. Note that this deduction is computed based on the voting `power` the misbehaving validator had at the height of the equivocation. As a result of the tokens deduction, 
+the [tokens per share](https://docs.cosmos.network/v0.47/modules/staking#delegator-shares)
+reduce and hence later on, when delegators undelegate or redelegate, the delegators retrieve back less
+tokens, effectively having their tokens slashed. This approach of slashing delegations does not utilize the
+`infractionHeight` in any way and hence the following scenario could occur:
```

Comment:
This behaviour is standard in the SDK, right? If so, could you make that more explicit, maybe link a place where this behaviour is stated? It seems like you are describing a flaw in the SDK, so it would be good to have more context, preferably just a link to the SDK docs about this

Reply:
Yes, this is standard SDK behaviour. I'm not aware of any SDK documentation clearly specifying this behaviour or such a scenario. The best I could find is the link already provided in regards to [delegator shares](https://docs.cosmos.network/v0.47/modules/staking#delegator-shares) where it's written:

> The reason for this mechanism is to simplify the accounting around slashing. Rather than iteratively slashing the tokens of every delegation entry, instead the Validators total bonded tokens can be slashed, effectively reducing the value of each issued delegator share.

Reply:
Thanks, that works for me!